main.go

```diff
@@ -328,3 +357,29 @@ func RollbackDeployment(w http.ResponseWriter, r *http.Request, _ httprouter.Par
 	responseByte, _ := json.Marshal(errs)
 	w.Write(responseByte)
 }
+
+func GetServices(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+
+	clusters := r.URL.Query().Get("clusters")
+	clusterList := strings.Split(clusters, ",")
+
+	services, err := clusterConnection.GetServices(clusterList)
+	
+	servicesClusterArray := make([]GetServicesResponse, len(clusterList))
+	for index, cluster := range clusterList {			
+		servicesClusterArray[index].Clusters  = cluster
+		servicesClusterArray[index].Services  = services[cluster]	
+		servicesClusterArray[index].Error  = err[index]		
+		
+	}
+
+	responseBytes, errors := json.MarshalIndent(servicesClusterArray, "", " ")
+	if errors != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return 
+	}
+
+	w.Write(responseBytes)
+	w.WriteHeader(http.StatusOK)
```

Comment:
WriteHeader should be called before Write call. Also in this case, we do not need to send StatusOK explicitly. Write already adds it.

Reply:
okay, i will fix it.